### ISSUE TITLE
[Design Card] 2023 media

### ISSUE THREAD

**Issue by aeiras** [microprofile/microprofile-marketing#754]
@rstjames, 

As discussed on today's marketing call, please create and share a **few card designs** that will welcome feedback and help shape the visual media in 2023. Thanks!

**Comment by aeiras** [comment]
Ok, so last week's Community call was quite helpful. 
The spreadsheet is updated to match the feedback as follows:

[2023 Consecutive Yearly Media Messages](https://docs.google.com/spreadsheets/d/1-j0SmFHSRw4pXMsdsY1x0R2HffIUZ3SQni_5zpoBdE8/edit#gid=1254202074)

1.  Technical call will have a quarterly reminder instead of day of the call reminder
2. Proactive news about each API will become the norm, tickets will be feed by MicroProfilers who are active on APIs. Specially when call-2-Action is required. 

---

@rstjames, 

Please provide a few design cards for 2023 media to enable feedback. Thank you!
The tickets for the few yearly messages will be created **only after** the new design is chosen.

**Comment by m-reza-rahman** [comment]
@rstjames to provide a few versions by Wednesday.

**Comment by rstjames** [comment]
Postponed to Monday

**Comment by aeiras** [comment]
> Postponed to Monday

sounds great. the new tickets will be created after the design card is chosen. :)

**Comment by aeiras** [comment]
@rstjames any update on the drafts? I will create the tickets today but no work can start before the new design is checked.

**Comment by aeiras** [comment]
@rstjames any updates on this?

**Comment by rstjames** [comment]
Option 1: Here are 3 rough layouts of sessions, call, and new members:

<img width="797" alt="Screenshot 2023-02-13 at 18 53 02" src="https://user-images.githubusercontent.com/12132146/218535111-9c9591f5-aa1c-4ba1-9b8a-6f92b37df3c0.png">
[Uploading Screenshot 2023-02-13 at 18.52.54.png…]()
 
<img width="798" alt="Screenshot 2023-02-13 at 18 53 10" src="https://user-images.githubusercontent.com/12132146/218535091-f14dfbab-5d88-4af6-a379-9367570dfb7c.png">

<img width="796" alt="Screenshot 2023-02-13 at 18 52 54" src="https://user-images.githubusercontent.com/12132146/218535397-ca3f9cc3-6bc7-4d99-b009-1a5f6e2f071f.png">

**Comment by aeiras** [comment]
I like the lighter version for 2023 social media, @rstjames. Thank you for sharing the drafts to enable feedback, you rock!

My recommendation is to send this ticket to the marketing mailing list and maybe ask for 72hrs feedback to close it at the end of Wednesday PDT so that Th and Fr can be used for closing the media tickets pending and the others that still need to be created for 2023 media. This task doesn't need to wait until the next marketing call to be completed, feedback via ticket is better IMO.

**Comment by rstjames** [comment]
Working on the 2nd option. Just not able to get it the way I like yet... I will send an email tomorrow 
@aeiras

**Comment by aeiras** [comment]
great!

**Comment by rstjames** [comment]
Version #2. Not full developed but has potential.  Personally prefer version 1.

<img width="595" alt="Screenshot 2023-02-15 at 20 18 42" src="https://user-images.githubusercontent.com/12132146/219134201-21099533-c107-4de3-bdf8-518a91a86af4.png">


<img width="597" alt="Screenshot 2023-02-15 at 20 18 31" src="https://user-images.githubusercontent.com/12132146/219134193-b0141db6-2231-4213-bff7-fb4de3a70756.png">

**Comment by aeiras** [comment]
I think we could use the 2 designs, @rstjames. I like both of them. The idea that MP social media can use its MP  design on 2nd designs is fantastic. Actually, it is very cool. 

Can you refine both of them and place them to be used interchangeably? That is if the majority of the group agrees!!!

**Comment by rstjames** [comment]
I rather not try to combine them. One has more straight edges and one is more curves. 
We can do a lot more variations with version 1 and it matches our last report.

**Comment by m-reza-rahman** [comment]
I am quite happy to go with Ryan’s recommendation.

**Comment by aeiras** [comment]
@rstjames, 

Any option that will increase MP media diversity is the winner. +1 in agreement with Reza. 
Let's drop the 2nd version in the 2024 design folder so that the nice design could be developed next year instead of starting from zero. 

It is Feb 16th, let's tackle setting up social media with 1. Finally, I can create the quarterly events git issues so you can tackle all those tickets at once with general media scheduling. Thank you for the good work on this!

**Comment by rstjames** [comment]
The 2nd version was added to the drive for next year.
